File: apps/Applications/ios/Local For Vocal/Local For Vocal/Sources/Views/Pages/HelpCenterView.swift
import SwiftUI

struct HelpCenterView: View {
    @Environment(\.presentationMode) var presentationMode

    struct FAQItem: Identifiable {
        let id = UUID()
        let question: String
        let answer: String
    }

    let faqs = [
        FAQItem(
            question: "How do I track my order?",
            answer: "You can track your order status in the 'My Orders' section of your account."),
        FAQItem(
            question: "What is the return policy?",
            answer:
                "We accept returns within 7 days of delivery for most items. Please check the product page for specific return details."
        ),
        FAQItem(
            question: "How can I contact customer support?",
            answer: "You can reach us at support@localforvocal.com or call our helpline."),
        FAQItem(
            question: "Do you offer international shipping?",
            answer: "Currently, we only ship within India."),
        FAQItem(
            question: "How do I change my password?",
            answer: "Go to Account Settings > Edit Profile to update your password."),
    ]

    var body: some View {
        VStack(spacing: 0) {
            // Header
            HStack {
                Button(action: {
                    presentationMode.wrappedValue.dismiss()
                }) {
                    Image(systemName: "arrow.left")
                        .font(.system(size: 20, weight: .medium))
                        .foregroundColor(Color(hex: "#1F2937"))
                }

                Spacer()

                Text("Help Center")
                    .font(.system(size: 18, weight: .bold))
                    .foregroundColor(Color(hex: "#1F2937"))

                Spacer()

                Color.clear.frame(width: 20, height: 20)
            }
            .padding(.horizontal, 16)
            .padding(.vertical, 14)
            .background(Color.white)
            .overlay(
                Rectangle()
                    .fill(Color(hex: "#E5E7EB"))
                    .frame(height: 1),
                alignment: .bottom
            )

            ScrollView {
                VStack(spacing: 16) {
                    ForEach(faqs) { faq in
                        DisclosureGroup(faq.question) {
                            Text(faq.answer)
                                .font(.system(size: 14))
                                .foregroundColor(.gray)
                                .padding(.top, 4)
                                .frame(maxWidth: .infinity, alignment: .leading)
                        }
                        .accentColor(.black)
                        .padding()
                        .background(Color.white)
                        .cornerRadius(8)
                        .shadow(color: .black.opacity(0.05), radius: 2, x: 0, y: 1)
                    }
                }
                .padding(16)
            }
            .background(Color(hex: "#F3F4F6"))
        }
        .navigationBarHidden(true)
    }
}

File: apps/Applications/android/app/src/main/java/com/localforvocalstartup/app/ui/pages/HelpCenterView.kt
package com.localforvocalstartup.app.ui.pages

import androidx.compose.animation.animateContentSize
import androidx.compose.foundation.background
import androidx.compose.foundation.clickable
import androidx.compose.foundation.layout.*
import androidx.compose.foundation.lazy.LazyColumn
import androidx.compose.foundation.lazy.items
import androidx.compose.foundation.shape.RoundedCornerShape
import androidx.compose.material.icons.Icons
import androidx.compose.material.icons.filled.ArrowBack
import androidx.compose.material.icons.filled.KeyboardArrowDown
import androidx.compose.material.icons.filled.KeyboardArrowUp
import androidx.compose.material3.*
import androidx.compose.runtime.*
import androidx.compose.ui.Alignment
import androidx.compose.ui.Modifier
import androidx.compose.ui.draw.shadow
import androidx.compose.ui.graphics.Color
import androidx.compose.ui.text.font.FontWeight
import androidx.compose.ui.unit.dp
import androidx.compose.ui.unit.sp
import com.localforvocalstartup.app.data.manager.NavigationManager

data class FAQItem(val question: String, val answer: String)

@OptIn(ExperimentalMaterial3Api::class)
@Composable
fun HelpCenterView() {
    val faqs = listOf(
        FAQItem("How do I track my order?", "You can track your order status in the 'My Orders' section of your account."),
        FAQItem("What is the return policy?", "We accept returns within 7 days of delivery for most items. Please check the product page for specific return details."),
        FAQItem("How can I contact customer support?", "You can reach us at support@localforvocal.com or call our helpline."),
        FAQItem("Do you offer international shipping?", "Currently, we only ship within India."),
        FAQItem("How do I change my password?", "Go to Account Settings > Edit Profile to update your password.")
    )

    Scaffold(
        topBar = {
            TopAppBar(
                title = { Text("Help Center", fontSize = 18.sp, fontWeight = FontWeight.Bold, color = Color(0xFF1F2937)) },
                navigationIcon = {
                    IconButton(onClick = { NavigationManager.goBack() }) {
                        Icon(Icons.Default.ArrowBack, contentDescription = "Back", tint = Color(0xFF1F2937))
                    }
                },
                colors = TopAppBarDefaults.topAppBarColors(containerColor = Color.White)
            )
        },
        containerColor = Color(0xFFF3F4F6)
    ) { padding ->
        LazyColumn(
            contentPadding = PaddingValues(16.dp),
            verticalArrangement = Arrangement.spacedBy(16.dp),
            modifier = Modifier.fillMaxSize().padding(padding)
        ) {
            items(faqs) { faq ->
                FAQCard(faq)
            }
        }
    }
}

@Composable
fun FAQCard(faq: FAQItem) {
    var expanded by remember { mutableStateOf(false) }

    Card(
        shape = RoundedCornerShape(8.dp),
        colors = CardDefaults.cardColors(containerColor = Color.White),
        elevation = CardDefaults.cardElevation(defaultElevation = 2.dp),
        modifier = Modifier.fillMaxWidth().animateContentSize()
    ) {
        Column(
            modifier = Modifier
                .clickable { expanded = !expanded }
                .padding(16.dp)
        ) {
            Row(
                verticalAlignment = Alignment.CenterVertically,
                horizontalArrangement = Arrangement.SpaceBetween,
                modifier = Modifier.fillMaxWidth()
            ) {
                Text(
                    text = faq.question,
                    fontSize = 16.sp,
                    fontWeight = FontWeight.Medium,
                    color = Color.Black,
                    modifier = Modifier.weight(1f)
                )
                Icon(
                    imageVector = if (expanded) Icons.Default.KeyboardArrowUp else Icons.Default.KeyboardArrowDown,
                    contentDescription = null,
                    tint = Color.Black
                )
            }
            if (expanded) {
                Spacer(modifier = Modifier.height(8.dp))
                Text(
                    text = faq.answer,
                    fontSize = 14.sp,
                    color = Color.Gray
                )
            }
        }
    }
}
